utils/config/config_test.go

```diff
@@ -87,9 +89,12 @@ func TestConvertConfigV1ToV2(t *testing.T) {
 
 	content, err := convertIfNeeded([]byte(config))
 	assert.NoError(t, err)
-	configV2 := new(ConfigV2)
-	assert.NoError(t, json.Unmarshal(content, &configV2))
-	assertionHelper(t, configV2, 2, false)
+	configV3 := new(ConfigV3)
+	assert.NoError(t, json.Unmarshal(content, &configV3))
+	assertionHelper(t, configV3, cliutils.GetConfigVersion(), false)
+
+	// The conversion to version 3, turns the Artifactory usernames to lowercase.
+	assert.Equal(t, configV3.Artifactory[0].User, "user")
```

Comment:
configV3.Artifactory[0].User, "user" should be swapped in places (the api is expected, actual)